sdk/resourcemanager/sql/armsql/CHANGELOG.md

```diff
@@ -1,5 +1,255 @@
 # Release History
 
+## 2.0.0-beta.1 (2023-07-28)
+### Breaking Changes
+
+- Function `*ServerDevOpsAuditSettingsClient.BeginCreateOrUpdate` parameter(s) have been changed from `(context.Context, string, string, string, ServerDevOpsAuditingSettings, *ServerDevOpsAuditSettingsClientBeginCreateOrUpdateOptions)` to `(context.Context, string, string, DevOpsAuditingSettingsName, ServerDevOpsAuditingSettings, *ServerDevOpsAuditSettingsClientBeginCreateOrUpdateOptions)`
+- Function `*ServerDevOpsAuditSettingsClient.Get` parameter(s) have been changed from `(context.Context, string, string, string, *ServerDevOpsAuditSettingsClientGetOptions)` to `(context.Context, string, string, DevOpsAuditingSettingsName, *ServerDevOpsAuditSettingsClientGetOptions)`
+- Type of `ManagedDatabaseRestoreDetailsProperties.NumberOfFilesDetected` has been changed from `*int64` to `*int32`
+- Type of `ManagedDatabaseRestoreDetailsProperties.PercentCompleted` has been changed from `*float64` to `*int32`
+- Type of `ManagedDatabaseRestoreDetailsProperties.UnrestorableFiles` has been changed from `[]*string` to `[]*ManagedDatabaseRestoreDetailsUnrestorableFileProperties`
+- Type of `ServerPrivateEndpointConnection.Properties` has been changed from `*PrivateEndpointConnectionProperties` to `*ServerPrivateEndpointConnectionProperties`
```

Comment:
Is this expected breaking change or it's caused by directive renaming?

Reply:
generated. https://github.com/Azure/azure-rest-api-specs/blob/d0d161eb19d5efd1c3d23d47a5491a53f41cbd7c/specification/sql/resource-manager/Microsoft.Sql/preview/2022-11-01-preview/ManagedDatabaseRestoreDetails.json#L158